arch/x86/pagetables.c

```diff
@@ -30,6 +30,8 @@
 #include <spinlock.h>
 #include <string.h>
 
+#define IA32_PAT 0x277
```

Comment:
Let's have `msr` header file (somewhere in arch/x86) where we insert these architectural msr defs

Reply:
We already have the processor.h